dist/chota.css

```diff
@@ -340,6 +340,7 @@ button {
   -webkit-transform: scale(1);
           transform: scale(1);
   display: inline-block;
+  cursor: pointer;
```

Comment:
not sure why this needs to be set explicitly. doesn't it work for your by default?

Reply:
No. Only `a` tag have cursor: pointer. You can see it here: [https://cdn.rawgit.com/jenil/chota/master/test/index.html#forms__action](https://cdn.rawgit.com/jenil/chota/master/test/index.html#forms__action)